### PR TITLE
Review and Update Polyx doc

### PR DESCRIPTION
**Pull request by lread**
Add sections and toc.

Move note and caution to top of doc.

Optimize flow and add some images on how to get the sha from GitHub. Explain how to get shas using git from command line.

Turf code block with ANSI escape codes and explain in text instead.

Link to Systems docs for overview image examples.

Edits for clarity.

### CODE REVIEW

**Comment by lread** [comment]
@tengstrand, ready for review and feedback.
As always, please read commit message before reviewing.

File: doc/polyx.adoc [hunk 1]
```diff
@@ -1,75 +1,108 @@
...
+==== Stable
 
-Now let's start a shell, using the `polyx` tool, by executing this from the workspace root:
+For the `sha` of the latest stable release, navigate to https://github.com/polyfy/polylith/releases/latest (versions will differ from images below, but the page layout should be the same):
 
-[source,shell]
-----
-clojure -M:polyx
-----
+. click on the tag: +
```
Comment (tengstrand) [inline doc/polyx.adoc:46]: Nice improvement!

File: doc/polyx.adoc [hunk 2]
```diff
@@ -1,75 +1,108 @@
...
+ANSI escape codes are not appropriate for some usages.
+Specify `color-mode:none` for plaintext output:
 
 [source,shell]
 ----
-poly info out:info.txt color-mode:none
+example$ info out:info.txt color-mode:none
 ----
 
 [source,shell]
```
Comment (tengstrand) [inline doc/polyx.adoc:108]: Change to [source,text].

**Review by tengstrand — CHANGES_REQUESTED** [review]

File: doc/polyx.adoc (hunk 1 above)
Comment (lread) [reply doc/polyx.adoc:46]: Tx!

File: doc/polyx.adoc (hunk 2 above)
Comment (lread) [reply doc/polyx.adoc:108]: Could do, but seems to render quite nicely on cljdoc:

image

Still want the change?

**Review by tengstrand — APPROVED** [review]

**Pull request merged**